code/modules/mob/living/simple_animal/hostile/wolf.dm

```diff
@@ -319,15 +319,15 @@
 			to_chat(user, "<span class='info'>It seems friendly to you.</span>")
 		var/remaining_health_percent = round((health/maxHealth)*100)
 		switch(remaining_health_percent)
-			if(100 to 60)
+			//if(60 to 100)
 				//Do nuthin
-			if(59 to 30)
+			if(30 to 59)
 				to_chat(user, "<span class='warning'>It seems quite hurt.</span>")
-			if(29 to 5)
+			if(5 to 29)
 				to_chat(user, "<span class='warning'>It seems extremely hurt.</span>")
-			if(5 to 1)
+			if(1 to 5)
 				to_chat(user, "<span class='warning'>It seems close to death!</span>")
-			if(0 to -INFINITY)
+			if(-INFINITY to 0)
 				to_chat(user, "<span class='warning'>It seems, well, dead.</span>")
```

Comment:
Why not just 'It is dead'?

Reply:
I blame Celt
Blame says Dragonbro (probably) but Celt probably actually wrote it